store/splus.js

```diff
@@ -29,6 +29,18 @@ export function customScheduleToRoute(customSchedule) {
   return { name: 'schedule', params, query };
 }
 
+export function shortenScheduleDegree(schedule) {
```

Comment:
Mach das mal 15 Zeilen weiter unten statt in jede Komponente 🐱 
like
```javascript
(schedule) => ({
  ...schedule,
  path: …,
  degreeShort: shortenScheduleDegree(schedule)
})
```
Dann kann man direkt darauf zugreifen und es wird nur einmal berechnet

Reply:
gute Idee